### ISSUE TITLE
Errors in log when exiting sway

### ISSUE THREAD

**Issue by panaman67** [swaywm/sway#6537]
When shutting down or exiting sway, the following errors are always shown next to each other and in this order:
`[wlr] [backend/drm/drm.c:1594] drmHandleEvent failed`
`[common/ipc-client.c:87] Unable to receive IPC response`

Is this normal? Please let me know if there is anything else I can provide. It seems to not be affected by the sway config.

### Please fill out the following:
- **Sway Version:**
  - `swaymsg -t get_version` or `sway -v`
  - sway version 1.6.1

- **Description:**
  - The steps you took in plain English to reproduce the problem.
  - 1. Start sway
  - 2. Exit sway to tty
  - 3. observe error messages

**Comment by emersion** [comment]
Please try Sway master.

**Comment by davidedellagiustina** [comment]
Same problem is happening to me. Version 1.8 instaled from `sway-git` on AUR.

**Comment by devgioele** [comment]
I confirm the presence of this error message on the master version 1.7

**Comment by houseofleaf** [comment]
The `[common/ipc-client.c:87] Unable to receive IPC response` error message is still present in sway 1.10.1. It appears each time sway is exited.

**Comment by houseofleaf** [comment]
Updating to say that the $mod+Shift+e exit button is a `swaynag` wrapper for `swaymsg exit`. When I start `sway` and manually execute `swaymsg exit`, the following output is printed in the TTY:

`Environment variable $DISPLAY not set, ignoring.
Environment variable $XDG_CURRENT_DESKTOP not set, ignoring.
2025-03-25 18:22:06 - [swaybg-1.2.1/main.c:282] Found config * for output eDP-1
warn: wayland.c:1728: compositor does not implement the XDG toplevel icon protocol
err: wayland.c:1611: failed to read events from the Wayland socket: Broken pipe
err: wayland.c:2247: failed to roundtrip Wayland display: Broken pipe
err: wayland.c:2247: failed to roundtrip Wayland display: Broken pipe
err: wayland.c:2247: failed to roundtrip Wayland display: Broken pipe
i3status: exiting due to signal.
warn: terminal.c:2015: slave exited with signal 1 (Hangup)
err: wayland.c:2212: failed to flush wayland socked: Broken pipe`

**Comment by houseofleaf** [comment]
Another update. By changing the keybind to `bindsym $mod+Shift+e exit` in my sway config, the error messages completely disappear. I don't know what caused the errors or why.

**Comment by emersion** [comment]
Likely swaynag?
